.forge-snapshots/poolManager bytecode size.snap

```diff
@@ -1 +1 @@
-24476
\ No newline at end of file
+24339
```

Comment:
decent progress, but still above 24k

Reply:
This saves 137 bytes. For comparison, using the internal function `checkNotDelegateCall` for `noDelegateCall` saves 149 bytes.